internal/controller/stackstatereconciler.go

```diff
@@ -1,7 +1,7 @@
 package controller
 
 import (
-	v1beta2types "github.com/docker/compose-on-kubernetes/api/compose/v1beta2"
+	latest "github.com/docker/compose-on-kubernetes/api/compose/v1alpha3"
```

Comment:
I'm wondering if there is another way?
Maybe add a latest package, and inside add an alias to the latest type.
Something like this:
```go
package latest

import(
"github.com/docker/compose-on-kubernetes/api/compose/v1alpha3"
)

type Stack = v1alpha3.Stack
```
So now you can import the latest stack everywhere, and it won't be impacted next time you will bump the stack version.

Reply:
I like it very much